api/client/credentials_test.go

```diff
@@ -23,127 +23,207 @@ import (
 	"path/filepath"
 	"testing"
 
+	"github.com/gravitational/teleport/api/sshutils"
+
 	"github.com/stretchr/testify/require"
+	"golang.org/x/crypto/ssh"
 )
 
 func TestLoadTLS(t *testing.T) {
 	t.Parallel()
 
 	// Load expected tls.Config.
-	expectedConfig, err := getExpectedConfig(t)
-	require.NoError(t, err)
+	expectedTLSConfig := getExpectedTLSConfig(t)
 
-	// Load and build tls.Config.
-	config, err := LoadTLS(expectedConfig).Config()
-	require.NoError(t, err)
+	// Load TLSConfigCreds.
+	creds := LoadTLS(expectedTLSConfig)
 
-	// Compare built and expected tls.Config.
-	require.Equal(t, config.Certificates, expectedConfig.Certificates)
-	require.Equal(t, config.RootCAs.Subjects(), expectedConfig.RootCAs.Subjects())
+	// Build tls.Config and compare to expected tls.Config.
+	tlsConfig, err := creds.TLSConfig()
+	require.NoError(t, err)
+	require.Equal(t, tlsConfig.Certificates, expectedTLSConfig.Certificates)
+	require.Equal(t, tlsConfig.RootCAs.Subjects(), expectedTLSConfig.RootCAs.Subjects())
```

Comment:
[`require.Equal`](https://pkg.go.dev/github.com/stretchr/testify/require#Equal) takes the expected value before the actual value, i.e. `expectedTLSConfig` should be the second argument here. The same applies to other such calls below.

Keeping the correct order helps with understanding test failure reports.